homeassistant/components/openhome/strings.json

```diff
@@ -1,12 +1,12 @@
 {
   "services": {
     "invoke_pin": {
-      "name": "Invoke PIN",
-      "description": "Invokes a PIN on the specified device.",
+      "name": "Play pin",
+      "description": "Starts playing content pinned on the specified device.",
       "fields": {
         "pin": {
-          "name": "PIN",
-          "description": "Which PIN to invoke."
+          "name": "Pin",
```

Comment:
Is it name, number or id here?

Reply:
Here's a picture of the Linn app where users can pin items. I'd be inclined to use "Pin" here

![image](https://github.com/user-attachments/assets/102d877e-4de4-43c7-949f-7c68bd1fa511)

Reply:
Maybe "Index" - since the card has already mentioned Pin?

Reply:
Maybe "Pin Id"?